cfme/configure/configuration/__init__.py

```diff
@@ -1555,6 +1557,12 @@ def update(self, updates, cancel=False, samba_validate=False):
         else:
             form_buttons.save()
 
+    @property
+    def last_date(self):
```

Comment:
We need to make sure, that we are on the right page, when trying to fetch data from the table.